### PR TITLE
Add kubelet-density light

### PR DESCRIPTION
**Pull request by rsevilla87**
We need kubelet-density light to establish a pass/fail criteria based on the pod startup latency.

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: jjb/dynamic/kubelet-density-light.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        # fetch the kubeconfig from the orchestration host
+        echo "Fetching the kubeconfig from the orchestration host"
+        KUBECONFIG_FILE=${KUBECONFIG_FILE:-~/.kube/config}
+        scp ${OPTIONS} -i ${PRIVATE_KEY} ${ORCHESTRATION_USER}@${ORCHESTRATION_HOST}:${KUBECONFIG_FILE} ${WORKSPACE}/kubeconfig
+        export KUBECONFIG=${WORKSPACE}/kubeconfig
+
+        # run tests
+        export TERM=screen-256color
+        pushd workloads/kube-burner
+        ./run_clusterdensity_test_fromgit.sh
```
Comment (chaitanyaenr) [inline jjb/dynamic/kubelet-density-light.yml:24]: s/run_kubeletdensity_test_fromgit.sh/run_kubeletdensity_test_fromgit.sh

File: jjb/dynamic/kubelet-density-light.yml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    concurrent: true
+    description: |
+      Cluster-density-light tworkload is used to measure control plane density for OpenShift. This test loads several objects across the cluster using cluster loader.
```
Comment (chaitanyaenr) [inline jjb/dynamic/kubelet-density-light.yml:27]: s/cluster-density/kubelet-density

Comment (rsevilla87) [reply jjb/dynamic/kubelet-density-light.yml:27]: good catch!

File: Jenkinsfile [hunk 3]
```diff
@@ -14,6 +14,7 @@ def ocpv3_scale = OPENSHIFTv3_SCALE.toString().toUpperCase()
 def ocpv4_scale = OPENSHIFTv4_SCALE.toString().toUpperCase()
 def cluster_density = CLUSTER_DENSITY.toString().toUpperCase()
 def kubelet_density = KUBELET_DENSITY.toString().toUpperCase()
+def kubelet_density_light = KUBELET_DENSITY_LIGTH.toString().toUpperCase()
```
Comment (chaitanyaenr) [inline Jenkinsfile:17]: s/KUBELET_DENSITY_LIGTH/KUBELET_DENSITY_LIGHT

Comment (rsevilla87) [reply Jenkinsfile:17]: fixed!

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**